README.md

```diff
@@ -1,28 +1,37 @@
-# Uprotocol C++ Library 
+# uProtocol C++ Interface Library (up-cpp)
 
 ## Welcome!
 
-This is the C++ library that extends up-core-api to provide serializers, validators, and language specific interface definitions for uProtocol. 
+This library provides the primary interface to uProtocol for uEntity
+applications. The primary point of interaction for most applications will be the
+layer 2 "communications" objects.
 
-*_IMPORTANT NOTE:_ This project is under active development*
-
-The core functional factory methods that make it more intuitive to create URIs that are used to configure source and sink (destination) elements in the uProtocol and additional helper methods.
+In addition to `up-cpp`, a uEntity will also need at least on transport
+implementation, such as [up-transport-zenoh-cpp](/eclipse-uprotocol/up-transport-zenoh-cpp).
 
+*_IMPORTANT NOTE:_ This project is under active development*
 
 ## Getting Started
 
-The library is built and packaged using conan dependency management. 
-
 ### Requirements:
 - Compiler: GCC/G++ 11 or Clang 13
 - Conan : 1.59 or latest 2.X
-- Ubuntu 22.04
+
+#### Conan packages
+
+Using the recipes found in [up-conan-recipes](/gregmedd/up-conan-recipes), build
+these Conan packages:
+
+1. [up-core-api](/eclipse-uprotocol/up-spec) - `conan create --version 1.5.8 --build=missing up-core-api/developer`
```

Comment:
Maybe note that this command is for Conan 2.x and that the Conan 1.x command is:
```
conan create up-core-api/developer 1.5.8@ --build=missing 
```